data/operators/amenity/telephone.json

```diff
@@ -45,20 +45,22 @@
     },
     {
       "displayName": "Bell",
```

Comment:
This entry and the one below are the same.
Everyone calls the company `Bell`, their real name is `Bell Canada`
I'm not sure how you would handle merging the 2 together so they both match